server/server_test.go

```diff
@@ -245,8 +245,12 @@ func TestGinPostCreateGame(t *testing.T) {
 		var game model.Game
 		readBody(t, w.Body, &game)
 		// verify the players are in the game
+		pIDs := make(map[model.PlayerID]struct{}, len(game.Players))
+		for _, p := range game.Players {
+			pIDs[p.ID] = struct{}{}
+		}
 		for _, pID := range cgr.PlayerIDs {
-			_, ok := game.PlayerColors[pID]
+			_, ok := pIDs[pID]
 			assert.True(t, ok)
 		}
```

Comment:
🔮 💡 we want to see that the game that was created has all of the players and no more, right? what about something like:

```go
require.Len(t, game.Players, len(cgr.PlayerIDs))
for _, p := range game.Players {
  assert.Contains(t, cgr.PlayerIDs, p.ID)
}
```